ERROR - 2024-11-13 16:34:50 --> mysqli_sql_exception: Unknown column 'property_number' in 'field list' in C:\laragon\www\dilg\backend\backend\system\Database\MySQLi\Connection.php:306
Stack trace:
#0 C:\laragon\www\dilg\backend\backend\system\Database\MySQLi\Connection.php(306): mysqli->query('SELECT `propert...', 0)
#1 C:\laragon\www\dilg\backend\backend\system\Database\BaseConnection.php(692): CodeIgniter\Database\MySQLi\Connection->execute('SELECT `propert...')
#2 C:\laragon\www\dilg\backend\backend\system\Database\BaseConnection.php(606): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT `propert...')
#3 C:\laragon\www\dilg\backend\backend\system\Database\BaseBuilder.php(1615): CodeIgniter\Database\BaseConnection->query('SELECT `propert...', Array, false)
#4 C:\laragon\www\dilg\backend\backend\system\Model.php(237): CodeIgniter\Database\BaseBuilder->get()
#5 C:\laragon\www\dilg\backend\backend\system\BaseModel.php(624): CodeIgniter\Model->doFindAll(0, 0)
#6 C:\laragon\www\dilg\backend\backend\app\Controllers\DatabasePPEController.php(1148): CodeIgniter\BaseModel->findAll()
#7 C:\laragon\www\dilg\backend\backend\system\CodeIgniter.php(919): App\Controllers\DatabasePPEController->searchPropertyNumber('MDO')
#8 C:\laragon\www\dilg\backend\backend\system\CodeIgniter.php(494): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\DatabasePPEController))
#9 C:\laragon\www\dilg\backend\backend\system\CodeIgniter.php(353): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\laragon\www\dilg\backend\index.php(86): CodeIgniter\CodeIgniter->run()
#11 {main}
CRITICAL - 2024-11-13 16:34:50 --> Unknown column 'property_number' in 'field list'
in SYSTEMPATH\Database\BaseConnection.php on line 646.
 1 SYSTEMPATH\Database\BaseBuilder.php(1615): CodeIgniter\Database\BaseConnection->query('SELECT `property_number`
FROM `inventoryppe`
WHERE `property_number` LIKE :property_number: ESCAPE \'!\'', [...], false)
 2 SYSTEMPATH\Model.php(237): CodeIgniter\Database\BaseBuilder->get()
 3 SYSTEMPATH\BaseModel.php(624): CodeIgniter\Model->doFindAll(0, 0)
 4 APPPATH\Controllers\DatabasePPEController.php(1148): CodeIgniter\BaseModel->findAll()
 5 SYSTEMPATH\CodeIgniter.php(919): App\Controllers\DatabasePPEController->searchPropertyNumber('MDO')
 6 SYSTEMPATH\CodeIgniter.php(494): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\DatabasePPEController))
 7 SYSTEMPATH\CodeIgniter.php(353): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 8 FCPATH\index.php(86): CodeIgniter\CodeIgniter->run()
ERROR - 2024-11-13 16:34:59 --> mysqli_sql_exception: Unknown column 'property_number' in 'field list' in C:\laragon\www\dilg\backend\backend\system\Database\MySQLi\Connection.php:306
Stack trace:
#0 C:\laragon\www\dilg\backend\backend\system\Database\MySQLi\Connection.php(306): mysqli->query('SELECT `propert...', 0)
#1 C:\laragon\www\dilg\backend\backend\system\Database\BaseConnection.php(692): CodeIgniter\Database\MySQLi\Connection->execute('SELECT `propert...')
#2 C:\laragon\www\dilg\backend\backend\system\Database\BaseConnection.php(606): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT `propert...')
#3 C:\laragon\www\dilg\backend\backend\system\Database\BaseBuilder.php(1615): CodeIgniter\Database\BaseConnection->query('SELECT `propert...', Array, false)
#4 C:\laragon\www\dilg\backend\backend\system\Model.php(237): CodeIgniter\Database\BaseBuilder->get()
#5 C:\laragon\www\dilg\backend\backend\system\BaseModel.php(624): CodeIgniter\Model->doFindAll(0, 0)
#6 C:\laragon\www\dilg\backend\backend\app\Controllers\DatabasePPEController.php(1148): CodeIgniter\BaseModel->findAll()
#7 C:\laragon\www\dilg\backend\backend\system\CodeIgniter.php(919): App\Controllers\DatabasePPEController->searchPropertyNumber('MDO')
#8 C:\laragon\www\dilg\backend\backend\system\CodeIgniter.php(494): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\DatabasePPEController))
#9 C:\laragon\www\dilg\backend\backend\system\CodeIgniter.php(353): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\laragon\www\dilg\backend\index.php(86): CodeIgniter\CodeIgniter->run()
#11 {main}
CRITICAL - 2024-11-13 16:34:59 --> Unknown column 'property_number' in 'field list'
in SYSTEMPATH\Database\BaseConnection.php on line 646.
 1 SYSTEMPATH\Database\BaseBuilder.php(1615): CodeIgniter\Database\BaseConnection->query('SELECT `property_number`
FROM `inventoryppe`
WHERE `property_number` LIKE :property_number: ESCAPE \'!\'', [...], false)
 2 SYSTEMPATH\Model.php(237): CodeIgniter\Database\BaseBuilder->get()
 3 SYSTEMPATH\BaseModel.php(624): CodeIgniter\Model->doFindAll(0, 0)
 4 APPPATH\Controllers\DatabasePPEController.php(1148): CodeIgniter\BaseModel->findAll()
 5 SYSTEMPATH\CodeIgniter.php(919): App\Controllers\DatabasePPEController->searchPropertyNumber('MDO')
 6 SYSTEMPATH\CodeIgniter.php(494): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\DatabasePPEController))
 7 SYSTEMPATH\CodeIgniter.php(353): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 8 FCPATH\index.php(86): CodeIgniter\CodeIgniter->run()
CRITICAL - 2024-11-13 17:15:02 --> There is no data to update.
in SYSTEMPATH\BaseModel.php on line 1724.
 1 SYSTEMPATH\BaseModel.php(1724): CodeIgniter\Database\Exceptions\DataException::forEmptyDataset('update')
 2 SYSTEMPATH\BaseModel.php(916): CodeIgniter\BaseModel->transformDataToArray([], 'update')
 3 SYSTEMPATH\Model.php(776): CodeIgniter\BaseModel->update([...], [])
 4 APPPATH\Controllers\DatabasePPEController.php(3124): CodeIgniter\Model->update('6', [])
 5 SYSTEMPATH\CodeIgniter.php(919): App\Controllers\DatabasePPEController->updateVerification('6')
 6 SYSTEMPATH\CodeIgniter.php(494): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\DatabasePPEController))
 7 SYSTEMPATH\CodeIgniter.php(353): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 8 FCPATH\index.php(86): CodeIgniter\CodeIgniter->run()
CRITICAL - 2024-11-13 17:15:03 --> There is no data to update.
in SYSTEMPATH\BaseModel.php on line 1724.
 1 SYSTEMPATH\BaseModel.php(1724): CodeIgniter\Database\Exceptions\DataException::forEmptyDataset('update')
 2 SYSTEMPATH\BaseModel.php(916): CodeIgniter\BaseModel->transformDataToArray([], 'update')
 3 SYSTEMPATH\Model.php(776): CodeIgniter\BaseModel->update([...], [])
 4 APPPATH\Controllers\DatabasePPEController.php(3124): CodeIgniter\Model->update('6', [])
 5 SYSTEMPATH\CodeIgniter.php(919): App\Controllers\DatabasePPEController->updateVerification('6')
 6 SYSTEMPATH\CodeIgniter.php(494): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\DatabasePPEController))
 7 SYSTEMPATH\CodeIgniter.php(353): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 8 FCPATH\index.php(86): CodeIgniter\CodeIgniter->run()
CRITICAL - 2024-11-13 17:16:01 --> Trying to access array offset on value of type null
in APPPATH\Controllers\DatabasePPEController.php on line 3101.
 1 APPPATH\Controllers\DatabasePPEController.php(3101): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Trying to access array offset on value of type null', 'C:\\laragon\\www\\dilg\\backend\\backend\\app\\Controllers\\DatabasePPEController.php', 3101)
 2 SYSTEMPATH\CodeIgniter.php(919): App\Controllers\DatabasePPEController->updateVerification('null')
 3 SYSTEMPATH\CodeIgniter.php(494): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\DatabasePPEController))
 4 SYSTEMPATH\CodeIgniter.php(353): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 5 FCPATH\index.php(86): CodeIgniter\CodeIgniter->run()
